### ISSUE TITLE
Add an "external" icon in Open site link

### ISSUE THREAD

**Issue by fcoveram** [Automattic/wp-calypso#100229]
## Where

1. Go to `/sites`
2. Click on the more menu in the site row
3. Click on the "Open site" link
4. See how it opens a new tab

![Options popover in sites dashboard](https://github.com/user-attachments/assets/bcac81d6-ad9d-4c71-afb9-dbc909630ea7)

## Problem

The link needs an external icon as it opens a "different" site in new tab. Although seeing the live version of the site in edit, the navigation and site structure is different.

**Comment by taipeicoder** [comment]
Same as the WP Admin link.

**Comment by okmttdhr** [comment]
I don’t think there’s a straightforward solution to this issue without making changes upstream in Core. The icons have also been removed from A4A: p5j4vm-4Z3-p2

I’ve created an issue to track this: https://github.com/WordPress/gutenberg/issues/69421

**Comment by taipeicoder** [comment]
@fcoveram is there a need to have those menu items? Their action button equivalents are right next to the menu.

![Image](https://github.com/user-attachments/assets/0dab4204-7549-4054-a665-089d34471b06)

**Comment by okmttdhr** [comment]
FYI, unfortunately, if we just define them as primary actions (displayed as icons), we have to include the items in the menu as well (No option to hide them without fragile overrides). This is one of the limitations imposed by Core. 🫠
https://github.com/Automattic/wp-calypso/blob/3b13df86567bf021f1fa8ac22b2e0c65dddf2f1c/client/sites/components/sites-dataviews/actions.tsx#L343

**Comment by okmttdhr** [comment]
Linking to the conversation starter in Slack: p1741226207962159-slack-C05QAFZSFTL

**Comment by jameskoster** [comment]
It was a design decision for the ellipsis menu to include _all_ actions so that it is consistent across DataViews layouts. Also because long term it will serve as a context menu on right-click.

Let's avoid overriding this core behavior if we can 🙏

**Comment by okmttdhr** [comment]
**Status**: On Hold

- We can proceed with the approach supporting prefix icons for primary actions: p1741574029340309/1741226207.962159-slack-C05QAFZSFTL
- Hold on for @\oandregal to have a way to advance DataViews on Dotcom: p1741599965491899/1741226207.962159-slack-C05QAFZSFTL
- Meanwhile, there's a patch in GB: https://github.com/WordPress/gutenberg/pull/69512

**Comment by taipeicoder** [comment]
We are aligned in that we want to avoid overriding core behavior. Moving this to "For Later" as this issue depends on changes upstream.
